messaging-tck/pom.xml

```diff
@@ -85,7 +85,7 @@
         <dependency>
             <groupId>jakarta.tck</groupId>
             <artifactId>jms-tck</artifactId>
-            <version>${jakarta.tck.messaging.version}</version>
+            <version>11.0.0-payara</version>
```

Comment:
probably we can use a property to reference the version used for the execution

Reply:
Done